packages/client/hmi-client/src/page/Home.vue

```diff
@@ -260,10 +260,10 @@ const projectsTabs = computed<{ title: string; projects: Project[] }[]>(() => [
 const columns = ref([
 	{ field: 'name', header: 'Project title' },
 	{ field: 'description', header: 'Description' },
-	{ field: 'username', header: 'Author' },
+	{ field: 'owner', header: 'Owner' },
```

Comment:
I'm not sure we want to change the header for `Owner` here?
Worth asking @pascaleproulx about it.

Reply:
changed back